pgpool2-4.6.yaml

```diff
@@ -63,6 +68,59 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
+    pipeline:
+      - name: "Symlink all pgpool binaries to /opt/pgpool-II/bin"
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/pgpool-II/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.subpkgdir}}/opt/pgpool-II/bin/${bin}"
+          done
+
+          # Link sample files
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc"
+          # Create symlinks from /etc/pgpool2/* to /opt/pgpool-II/etc/
+          for bin in \
+            aws_eip_if_cmd.sh.sample follow_primary.sh.sample pool_hba.conf.sample \
+            aws_rtb_if_cmd.sh.sample pcp.conf.sample recovery_1st_stage.sample escalation.sh.sample \
+            pgpool_remote_start.sample replication_mode_recovery_1st_stage.sample failover.sh.sample \
+            pgpool.conf.sample replication_mode_recovery_2nd_stage.sample \
+          ; do
+            ln -s "/etc/pgpool2/${bin}" "${{targets.subpkgdir}}/opt/pgpool-II/etc/${bin}"
+          done
+
+  - name: ${{package.name}}-oci-entrypoint
+    description: "Entrypoint and start scripts for pgpool images"
+    dependencies:
+      provides:
+        - pgpool2-oci-entrypoint=${{package.full-version}}
+      runtime:
+        - bash
+        - coreutils
+        - grep
+        - gawk
+        - gnutar
+    pipeline:
+      - name: "Fetch and add startup scripts"
+        runs: |
+          # Add start.sh and entrypoint scripts same as upstream
+          git clone --depth 1 --branch master https://github.com/pgpool/pgpool2_on_k8s
```

Comment:
We should be using `git-checkout` here, even if this is a subpackage, so that our packages are reproducible. I would split this into a separate package manifest personally so we can track updates

Reply:
I've updated the PR with git-checkout
The upstream image is no longer maintained and these entrypoint files got last update 3 years ago. 
https://github.com/pgpool/pgpool2_on_k8s/blob/master/pgpool.docker/entrypoint.sh
Fortunately, `pgpool2` behaves the same and these scripts work well.